### DISCUSSION TITLE
8/8 想問學長的問題（沒問完就寄信)

### DISCUSSION THREAD

**Discussion by Will150143** [ChiuAlfredo/options_day_data#43] [General]
8/8 想問學長的問題（沒問完就寄信)

**Comment by ChiuAlfredo** [comment]
星期一時，在已知可能會下跌，並且vix極高時，會怎麼去做策略計劃

**Comment by ChiuAlfredo** [comment]
![Screenshot 2024-08-06 at 12 04 59 PM](https://github.com/user-attachments/assets/bd83ff90-ee64-42fc-85bb-79897b0f50a8)
在做IC時，如果遇到買賣價差大，導致無法順利平倉，這樣的情況是很常見嗎。如果遇見了要如何處理

**Comment by Will150143** [comment]
在結算日最後一小時做交易時較常出現哪些狀況，各自對應的策略是什麼（開始想專題要做哪一個）
